src/plugins/i18n.js

```diff
@@ -3,10 +3,12 @@ import {
 } from 'vue-i18n'
 import en from '*/locales/en.json'
 import ru from '*/locales/ru.json'
+import it from '*/locales/it.json'
```

Comment:
@filipporomani Actually could you please place `it` after `en`?
Languages should go alphabetically.

Reply:
Sure

Reply:
Pusehd again! fixed